back/.rubocop.yml

```diff
@@ -1,96 +1,103 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 require:
   - rubocop-rails
   - rubocop-performance
   - rubocop-rspec
 
 AllCops:
-  DisabledByDefault: true
+  NewCops: enable
   Exclude:
   - 'db/schema.rb'
   - '**/*.builder'
   - '**/.*.*'
   - '**/bin/**/*'
+  - 'engines/ee/**/*'
+
+Style/Documentation:
+  # Most classes are not documented, so turn off this Cop.
+  Enabled: false
 
-Style/StringLiterals:
+Style/SymbolProc:
   Enabled: true
+  IgnoredMethods:
+    - attribute
+    - throttle
+  Exclude:
+    - 'db/migrate/**/*'
+
+Style/NumericPredicate:
+  # Turned off because `something == 0` and `something.zero?`
+  # are not the same if `something` is not a number.
+  Enabled: false
+
+Layout/HashAlignment:
+  # There are a lot of places where EnforcedColonStyle: table is expected.
+  # Discussion needed, because the default will affect a lot of table like hash literals.
```

Comment:
My 2 cents: I'd prefer to stay as close to the default Rubocop config as possible where we can and spare us these endless discussions and let an auto-formatter do that job for me instead.